src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -3125,6 +3127,17 @@ TEST_F(SQLTranslatorTest, CastStatement) {
     const auto [actual_lqp, translation_info] = sql_to_lqp_helper("SELECT CAST('2000-01-01' as DATE);");
     EXPECT_LQP_EQ(actual_lqp, expected_lqp);
   }
+  {
+    // BIGINT is an alias for LONG
+    const auto cast_expression = expression_vector(cast_(value_(int32_t{1}), DataType::Long));
+    // clang-format off
+    const auto expected_lqp =
+      ProjectionNode::make(cast_expression,
+        DummyTableNode::make());
+    // clang-format on
```

Comment:
```suggestion
    const auto expected_lqp =
    ProjectionNode::make(cast_expression,
      DummyTableNode::make());
    // clang-format on
```

Reply:
Hm, we are inconsistently regarding the spacing in this file. 
`\s{2}const auto expected_lqp.+\n[ ]{4}` --> 30 matches
`\s{2}const auto expected_lqp.+\n[ ]{2}` --> 143 matches

However, as mostly the unindented version is used, I'm going to change all occurrences according to your suggested change.